api/v1alpha1/edgedevice_types.go

```diff
@@ -53,6 +53,9 @@ type SystemMetricsConfiguration struct {
 	// AllowList defines name and namespace of a ConfigMap containing
 	// list of system metrics that should be scraped
 	AllowList *ObjectRef `json:"allowList,omitempty"`
+
+	// Disabled when set to true instructs the device to turn off system metrics collection
+	Disabled bool `json:"disabled,omitempty"`
```

Comment:
Just out of curiosity why using `disable` instead of `enable`? I think mostly there are `enable` options instead of `disable`.

Reply:
Metrics are enabled by default and `enabled` handling would be more cumbersome than `disabled`.